os_migrate/roles/import_flavors/meta/main.yml

```diff
@@ -14,3 +14,6 @@ galaxy_info:
   galaxy_tags: ["osmigrate"]
 dependencies:
   - role: os_migrate.os_migrate.prelude_dst
+  - role: os_migrate.os_migrate.validate_resource_files
+    vars: 
+      resource_file: flavors.yml 
```

Comment:
Looks good, i'd just perhaps make it a bit more generic in case we want to use the role elswhere:

* Accept multiple files (and name the param plural - `resource_files`).

* Do not assume the '{{ os_migrate_data_dir }}` prefix inside the role, but feed it from outside.

So invoking the role from here would then look like:

```
  - role: os_migrate.os_migrate.validate_resource_files
    vars: 
      resource_files:
        - "{{ os_migrate_data_dir }}/flavors.yml"
```

or:

```
  - role: os_migrate.os_migrate.validate_resource_files
    vars: 
      resource_files: ["{{ os_migrate_data_dir }}/flavors.yml"]
```

(I'm not sure if the latter variant passes lint conventions or not...)